pandas/core/generic.py

```diff
@@ -1940,6 +1940,84 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n = None, frac = None , replace = False, weights = None, seed = None):
```

Comment:
The PEP8 convention is to not use spaces around `=` when used in function signatures